angular.js

```diff
@@ -0,0 +1,345 @@
+/* eslint-env node */
+/* eslint-disable quote-props */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const spellcheckRule = require('./spell-check').rules['spellcheck/spell-checker'];
+
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    es6: true,
+  },
+  parser: 'babel-eslint',
+  parserOptions: {
+    sourceType: 'module',
+    'ecmaVersion': 2018,
+  },
+  plugins: [
+    'spellcheck',
+    'no-only-tests',
+  ],
+  rules: {
+    'spellcheck/spell-checker': (() => {
+      Array.prototype.push.apply(spellcheckRule[1].skipWords, [
+        'acc', // DXHotelsNetApi\Client\views\currentHotel.js ?
+        'accessor',
```

Comment:
this part is common for all frameworks, let's add this directly to `spell-checker.js`